docs/can-guides/commitment/migrating_to_3.0.md

```diff
@@ -225,6 +237,146 @@ If you were using `can.view.preload` then use [can-stache.registerPartial] inste
 stache.registerPartial("some-id", renderer);
 ```
 
+### API changes
+
+A number of API changes have been made in CanJS 3.
+
+#### can.event
+
+Some methods have been renamed in `can.event`.
+
+```js
+can.event.addEvent(el, 'click', function() {});
+can.event.removeEvent(el, 'click', function() {});
+```
+
+Becomes:
+
+```js
+can.event.addEventListener(el, 'click', function() {});
+can.event.removeEventListener(el, 'click', function() {});
+```
+
+#### can.batch
+
+`trigger` has been renamed to `dispatch`.
+
+```js
+can.batch.trigger(myObj, 'myEvent');
+```
+
+Becomes:
+
+```js
+import canBatch from 'can-event/batch/batch';
+canBatch.dispatch(myObj, 'myEvent');
+```
+
+#### can.extend
+
+This method has been split into two: a shallow and deep merge. Previously, passing true as the first parameter would do a deep merge. Now, you explicitly invoke the deep merge or shallow merge function.
+
+```js
+can.extend({}, { answer: 42 }); // shallow
+can.extend(true, {}, { answer: 42 }); // deep
+```
+
+Becomes:
+
+```js
+import assign from 'can-util/js/assign/assign';
+import deepAssign from 'can-util/js/deepAssign/deepAssign';
+assign({}, { answer: 42 }); // shallow
+deepAssign({}, { answer: 42 }); // deep
+```
+
+#### can.viewModel
+
+This method now expects a DOM element as the first parameter:
```

Comment:
`can-view-model` can take a string as the first parameter, it just isn't reflected in the docs. See https://github.com/canjs/can-view-model/blob/master/can-view-model.js#L9 and https://github.com/canjs/can-view-model/issues/19.